diff -Nru src/tcljx.compiler/module-info.java ../tcljx.compiler.patched/module-info.java
--- src/tcljx.compiler/module-info.java	2025-10-30 13:03:33.598942032 +0100
+++ ../tcljx.compiler.patched/module-info.java	2025-11-02 14:34:34.400379661 +0100
@@ -1,5 +1,5 @@
  module tcljx.compiler {
-   requires transitive tcljc.core;
+   requires transitive tcljx.alpha;
    
    // exports tcljx.nmspgen.classdata;
    exports tcljx.nmspgen.codegen;
diff -Nru src/tcljx.compiler/tcljx/data/config.cljt ../tcljx.compiler.patched/tcljx/data/config.cljt
--- src/tcljx.compiler/tcljx/data/config.cljt	2025-10-29 14:29:07.160521153 +0100
+++ ../tcljx.compiler.patched/tcljx/data/config.cljt	2025-11-02 14:34:34.398379642 +0100
@@ -7,7 +7,7 @@
 ;; You must not remove this notice, or any other, from this software.
 (ns tcljx.data.config
   (:require
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (java.lang.classfile ClassFile ClassBuilder)
    (java.lang.constant ClassDesc)))
diff -Nru src/tcljx.compiler/tcljx/data/type.cljt ../tcljx.compiler.patched/tcljx/data/type.cljt
--- src/tcljx.compiler/tcljx/data/type.cljt	2025-11-02 12:02:43.545500348 +0100
+++ ../tcljx.compiler.patched/tcljx/data/type.cljt	2025-11-02 14:34:34.398379642 +0100
@@ -1,5 +1,5 @@
 (ns tcljx.data.type
-  (:require [tinyclj.core :as cc])
+  (:require [clojure.core :as cc])
   (:import (java.lang.classfile CodeBuilder TypeKind)
            (java.lang.constant ConstantDesc)
            (java.lang.invoke MethodHandles$Lookup))
diff -Nru src/tcljx.compiler/tcljx/main/efmt.cljt ../tcljx.compiler.patched/tcljx/main/efmt.cljt
--- src/tcljx.compiler/tcljx/main/efmt.cljt	2025-10-21 15:21:33.281236323 +0200
+++ ../tcljx.compiler.patched/tcljx/main/efmt.cljt	2025-11-02 14:34:34.400379661 +0100
@@ -9,7 +9,7 @@
   (:require
    (tcljx.data [wrong :as wrong])
    [tcljx.alpha.ptest__style :as style]
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.wrong WrongInfo)
    (java.lang.constant ClassDesc ConstantDescs)
diff -Nru src/tcljx.compiler/tcljx/main/options.cljt ../tcljx.compiler.patched/tcljx/main/options.cljt
--- src/tcljx.compiler/tcljx/main/options.cljt	2025-10-25 17:20:38.186822962 +0200
+++ ../tcljx.compiler.patched/tcljx/main/options.cljt	2025-11-02 14:34:34.400379661 +0100
@@ -8,7 +8,7 @@
 (ns tcljx.main.options
   (:require
    (tcljx.data [config :as cfg] [files :as files])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (java.nio.file Path Files)))
 
diff -Nru src/tcljx.compiler/tcljx/main/watcher.cljt ../tcljx.compiler.patched/tcljx/main/watcher.cljt
--- src/tcljx.compiler/tcljx/main/watcher.cljt	2025-11-02 14:33:38.107834634 +0100
+++ ../tcljx.compiler.patched/tcljx/main/watcher.cljt	2025-11-02 14:34:57.109599460 +0100
@@ -7,7 +7,7 @@
 ;; You must not remove this notice, or any other, from this software.
 (ns tcljx.main.watcher
   (:require (tcljx.data [files :as files])
-            [tinyclj.string :as str])
+            [clojure.string :as str])
   (:import (java.net URI)
            (java.nio.file Path Paths FileSystems WatchService WatchKey
                           WatchEvent StandardWatchEventKinds)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/primitive.cljt ../tcljx.compiler.patched/tcljx/nmspgen/primitive.cljt
--- src/tcljx.compiler/tcljx/nmspgen/primitive.cljt	2025-11-01 12:22:02.498458607 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/primitive.cljt	2025-11-02 14:34:34.397379632 +0100
@@ -6,7 +6,7 @@
    (tcljx.nmspgen [pkggen :as pkggen] [constgen :as constgen] [emit :as emit]
                   [insn :as insn] [runtime :as rt] [condgen :as condgen]
                   [opgen :as opgen] [junctgen :as junctgen])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.type Wrapper)
    (tcljx.data.ir CodeEmitter Loadable Entity ImmediateEntity Constant Binding)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/publics.cljt ../tcljx.compiler.patched/tcljx/nmspgen/publics.cljt
--- src/tcljx.compiler/tcljx/nmspgen/publics.cljt	2025-10-29 22:15:35.885326528 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/publics.cljt	2025-11-02 14:34:34.397379632 +0100
@@ -5,7 +5,7 @@
                [code :as code] [ir :as ir])
    (tcljx.nmspgen [runtime :as rt] [insn :as insn] [constgen :as constgen]
                   [defgen :as defgen] [fngen :as fngen])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.ir Entity ImmediateEntity ArityType MethodSpec ProvidedNamespace)
    (tcljx.nmspgen.runtime Runtime)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/runtime.cljt ../tcljx.compiler.patched/tcljx/nmspgen/runtime.cljt
--- src/tcljx.compiler/tcljx/nmspgen/runtime.cljt	2025-10-29 21:59:19.021480426 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/runtime.cljt	2025-11-02 14:34:34.397379632 +0100
@@ -1,7 +1,7 @@
 (ns tcljx.nmspgen.runtime
   (:require
    (tcljx.data [config :as cfg] [files :as files] [ir :as ir])
-   [tinyclj.core :as cc])
+   [clojure.core :as cc])
   (:import
    (java.lang.constant ClassDesc MethodTypeDesc MethodHandleDesc ConstantDescs
                        DirectMethodHandleDesc$Kind)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/typegen.cljt ../tcljx.compiler.patched/tcljx/nmspgen/typegen.cljt
--- src/tcljx.compiler/tcljx/nmspgen/typegen.cljt	2025-10-31 12:33:08.621105104 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/typegen.cljt	2025-11-02 14:34:34.397379632 +0100
@@ -4,7 +4,7 @@
                [type :as type] [ir :as ir] [code :as code])
    (tcljx.nmspgen [util :as util] [runtime :as rt] env pkggen [emit :as emit]
                   [constgen :as constgen] [insn :as insn] [envgen :as envgen])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.ir CodeEmitter Entity MethodSpec FieldSpec NmspClass)
    (tcljx.nmspgen.env ClosureEntity Display StoredEnv)
diff -Nru src/tcljx.compiler/tcljx/parser/reader.cljt ../tcljx.compiler.patched/tcljx/parser/reader.cljt
--- src/tcljx.compiler/tcljx/parser/reader.cljt	2025-11-02 11:10:00.040899119 +0100
+++ ../tcljx.compiler.patched/tcljx/parser/reader.cljt	2025-11-02 14:34:34.400379661 +0100
@@ -8,7 +8,7 @@
 (ns tcljx.parser.reader
   (:require
    (tcljx.data [config :as cfg] [meta :as meta] [wrong :as wrong] [ir :as ir])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.ir Resolver FormReader)
    (tcljx.data.wrong WrongInfo)
diff -Nru src/tcljx.compiler/tcljx/parser/syntax.cljt ../tcljx.compiler.patched/tcljx/parser/syntax.cljt
--- src/tcljx.compiler/tcljx/parser/syntax.cljt	2025-10-21 15:21:33.118233547 +0200
+++ ../tcljx.compiler.patched/tcljx/parser/syntax.cljt	2025-11-02 14:34:34.400379661 +0100
@@ -1,6 +1,6 @@
 (ns tcljx.parser.syntax
   (:require (tcljx.data [config :as cfg] [wrong :as wrong])
-            [tinyclj.core :as cc])
+            [clojure.core :as cc])
   (:refer-clojure :exclude [vector list map symbol first second]))
 
 (defn info
