diff -Nru src/tcljx.compiler/module-info.java ../tcljx.compiler.patched/module-info.java
--- src/tcljx.compiler/module-info.java	2025-10-30 07:30:04.956009617 +0100
+++ ../tcljx.compiler.patched/module-info.java	2025-10-30 07:33:44.562658252 +0100
@@ -1,5 +1,5 @@
  module tcljx.compiler {
-   requires transitive tcljc.core;
+   requires transitive tcljx.alpha;
    
    // exports tcljx.nmspgen.classdata;
    exports tcljx.nmspgen.codegen;
diff -Nru src/tcljx.compiler/module-info.java.orig ../tcljx.compiler.patched/module-info.java.orig
--- src/tcljx.compiler/module-info.java.orig	1970-01-01 01:00:00.000000000 +0100
+++ ../tcljx.compiler.patched/module-info.java.orig	2025-10-30 07:33:25.525427976 +0100
@@ -0,0 +1,58 @@
+ module tcljx.compiler {
+   requires transitive tcljc.core;
+   
+   // exports tcljx.nmspgen.classdata;
+   exports tcljx.nmspgen.codegen;
+   exports tcljx.nmspgen.condgen;
+   exports tcljx.nmspgen.constgen;
+   exports tcljx.nmspgen.defgen;
+   exports tcljx.nmspgen.destdir;
+   exports tcljx.nmspgen.emit;
+   exports tcljx.nmspgen.entity;
+   exports tcljx.nmspgen.env;
+   exports tcljx.nmspgen.envgen;
+   exports tcljx.nmspgen.fngen;
+   exports tcljx.nmspgen.insn;
+   exports tcljx.nmspgen.macro;
+   exports tcljx.nmspgen.pkggen;
+   exports tcljx.nmspgen.primitive;
+   exports tcljx.nmspgen.publics;
+   exports tcljx.nmspgen.runtime;
+   exports tcljx.nmspgen.switchgen;
+   exports tcljx.nmspgen.trygen;
+   exports tcljx.nmspgen.typegen;
+   exports tcljx.nmspgen.universe;
+   exports tcljx.nmspgen.util;
+   exports tcljx.data.code;
+   exports tcljx.data.config;
+   exports tcljx.data.context;
+   exports tcljx.data.files;
+   exports tcljx.data.ir;
+   exports tcljx.data.member;
+   exports tcljx.data.meta;
+   exports tcljx.data.reflect;
+   exports tcljx.data.resolve;
+   exports tcljx.data.type;
+   exports tcljx.data.wrong;
+   exports tcljx.main;
+   exports tcljx.main.efmt;
+   exports tcljx.main.invoke;
+   exports tcljx.main.options;
+   exports tcljx.main.tclj0;
+   exports tcljx.main.tclj1;
+   exports tcljx.main.watcher;
+   exports tcljx.parser.constant;
+   exports tcljx.parser.def;
+   exports tcljx.parser.deftype;
+   exports tcljx.parser.fn;
+   exports tcljx.parser.form;
+   exports tcljx.parser.interop;
+   exports tcljx.parser.let;
+   exports tcljx.parser.loop;
+   exports tcljx.parser.namespace;
+   exports tcljx.parser.newobj;
+   exports tcljx.parser.parse;
+   exports tcljx.parser.reader;
+   exports tcljx.parser.syntax;
+   exports tcljx.parser.trycatch;
+}
diff -Nru src/tcljx.compiler/module-info.java.rej ../tcljx.compiler.patched/module-info.java.rej
--- src/tcljx.compiler/module-info.java.rej	1970-01-01 01:00:00.000000000 +0100
+++ ../tcljx.compiler.patched/module-info.java.rej	2025-10-30 07:33:25.525427976 +0100
@@ -0,0 +1,9 @@
+--- module-info.java	2025-10-29 22:26:12.231539787 +0100
++++ module-info.java	2025-10-29 22:26:37.901994636 +0100
+@@ -1,5 +1,5 @@
+  module tcljx.compiler {
+-   requires transitive tcljc.core;
++   requires transitive tcljx.alpha;
+ 
+    // exports tcljx.alpha.ptest__align;
+    // exports tcljx.alpha.ptest;
diff -Nru src/tcljx.compiler/tcljx/data/config.cljt ../tcljx.compiler.patched/tcljx/data/config.cljt
--- src/tcljx.compiler/tcljx/data/config.cljt	2025-10-29 14:29:07.160521153 +0100
+++ ../tcljx.compiler.patched/tcljx/data/config.cljt	2025-10-30 07:33:25.524427964 +0100
@@ -7,7 +7,7 @@
 ;; You must not remove this notice, or any other, from this software.
 (ns tcljx.data.config
   (:require
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (java.lang.classfile ClassFile ClassBuilder)
    (java.lang.constant ClassDesc)))
diff -Nru src/tcljx.compiler/tcljx/data/type.cljt ../tcljx.compiler.patched/tcljx/data/type.cljt
--- src/tcljx.compiler/tcljx/data/type.cljt	2025-10-26 14:33:00.368188966 +0100
+++ ../tcljx.compiler.patched/tcljx/data/type.cljt	2025-10-30 07:33:25.524427964 +0100
@@ -1,5 +1,5 @@
 (ns tcljx.data.type
-  (:require [tinyclj.core :as cc])
+  (:require [clojure.core :as cc])
   (:import (java.lang.classfile CodeBuilder TypeKind)
            (java.lang.constant ConstantDesc)
            (java.lang.invoke MethodHandles$Lookup))
diff -Nru src/tcljx.compiler/tcljx/main/efmt.cljt ../tcljx.compiler.patched/tcljx/main/efmt.cljt
--- src/tcljx.compiler/tcljx/main/efmt.cljt	2025-10-21 15:21:33.281236323 +0200
+++ ../tcljx.compiler.patched/tcljx/main/efmt.cljt	2025-10-30 07:33:25.525427976 +0100
@@ -9,7 +9,7 @@
   (:require
    (tcljx.data [wrong :as wrong])
    [tcljx.alpha.ptest__style :as style]
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.wrong WrongInfo)
    (java.lang.constant ClassDesc ConstantDescs)
diff -Nru src/tcljx.compiler/tcljx/main/options.cljt ../tcljx.compiler.patched/tcljx/main/options.cljt
--- src/tcljx.compiler/tcljx/main/options.cljt	2025-10-25 17:20:38.186822962 +0200
+++ ../tcljx.compiler.patched/tcljx/main/options.cljt	2025-10-30 07:33:25.525427976 +0100
@@ -8,7 +8,7 @@
 (ns tcljx.main.options
   (:require
    (tcljx.data [config :as cfg] [files :as files])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (java.nio.file Path Files)))
 
diff -Nru src/tcljx.compiler/tcljx/main/watcher.cljt ../tcljx.compiler.patched/tcljx/main/watcher.cljt
--- src/tcljx.compiler/tcljx/main/watcher.cljt	2025-10-21 15:21:33.270236136 +0200
+++ ../tcljx.compiler.patched/tcljx/main/watcher.cljt	2025-10-30 07:33:25.525427976 +0100
@@ -7,7 +7,7 @@
 ;; You must not remove this notice, or any other, from this software.
 (ns tcljx.main.watcher
   (:require (tcljx.data [files :as files])
-            [tinyclj.string :as str])
+            [clojure.string :as str])
   (:import (java.nio.file Path Paths FileSystems WatchService WatchKey
                           WatchEvent StandardWatchEventKinds)
            (java.util.concurrent TimeUnit)))
diff -Nru src/tcljx.compiler/tcljx/nmspgen/primitive.cljt ../tcljx.compiler.patched/tcljx/nmspgen/primitive.cljt
--- src/tcljx.compiler/tcljx/nmspgen/primitive.cljt	2025-10-29 16:04:59.919073913 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/primitive.cljt	2025-10-30 07:33:25.522427940 +0100
@@ -6,7 +6,7 @@
    (tcljx.nmspgen [pkggen :as pkggen] [constgen :as constgen] [emit :as emit]
                   [insn :as insn] [runtime :as rt] [condgen :as condgen]
                   [junctgen :as junctgen])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.type Wrapper)
    (tcljx.data.ir CodeEmitter Loadable Entity ImmediateEntity Constant)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/publics.cljt ../tcljx.compiler.patched/tcljx/nmspgen/publics.cljt
--- src/tcljx.compiler/tcljx/nmspgen/publics.cljt	2025-10-29 22:15:35.885326528 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/publics.cljt	2025-10-30 07:33:25.522427940 +0100
@@ -5,7 +5,7 @@
                [code :as code] [ir :as ir])
    (tcljx.nmspgen [runtime :as rt] [insn :as insn] [constgen :as constgen]
                   [defgen :as defgen] [fngen :as fngen])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.ir Entity ImmediateEntity ArityType MethodSpec ProvidedNamespace)
    (tcljx.nmspgen.runtime Runtime)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/runtime.cljt ../tcljx.compiler.patched/tcljx/nmspgen/runtime.cljt
--- src/tcljx.compiler/tcljx/nmspgen/runtime.cljt	2025-10-29 21:59:19.021480426 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/runtime.cljt	2025-10-30 07:33:25.522427940 +0100
@@ -1,7 +1,7 @@
 (ns tcljx.nmspgen.runtime
   (:require
    (tcljx.data [config :as cfg] [files :as files] [ir :as ir])
-   [tinyclj.core :as cc])
+   [clojure.core :as cc])
   (:import
    (java.lang.constant ClassDesc MethodTypeDesc MethodHandleDesc ConstantDescs
                        DirectMethodHandleDesc$Kind)
diff -Nru src/tcljx.compiler/tcljx/nmspgen/typegen.cljt ../tcljx.compiler.patched/tcljx/nmspgen/typegen.cljt
--- src/tcljx.compiler/tcljx/nmspgen/typegen.cljt	2025-10-29 18:04:31.674183386 +0100
+++ ../tcljx.compiler.patched/tcljx/nmspgen/typegen.cljt	2025-10-30 07:33:25.522427940 +0100
@@ -4,7 +4,7 @@
                [type :as type] [ir :as ir] [code :as code])
    (tcljx.nmspgen [util :as util] [runtime :as rt] env pkggen [emit :as emit]
                   [constgen :as constgen] [insn :as insn] [envgen :as envgen])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.ir CodeEmitter Entity MethodSpec FieldSpec NmspClass)
    (tcljx.nmspgen.env ClosureEntity Display StoredEnv)
diff -Nru src/tcljx.compiler/tcljx/parser/reader.cljt ../tcljx.compiler.patched/tcljx/parser/reader.cljt
--- src/tcljx.compiler/tcljx/parser/reader.cljt	2025-10-27 14:20:11.736263967 +0100
+++ ../tcljx.compiler.patched/tcljx/parser/reader.cljt	2025-10-30 07:33:25.525427976 +0100
@@ -8,7 +8,7 @@
 (ns tcljx.parser.reader
   (:require
    (tcljx.data [config :as cfg] [meta :as meta] [wrong :as wrong] [ir :as ir])
-   [tinyclj.string :as str])
+   [clojure.string :as str])
   (:import
    (tcljx.data.ir Resolver FormReader)
    (tcljx.data.wrong WrongInfo)
diff -Nru src/tcljx.compiler/tcljx/parser/syntax.cljt ../tcljx.compiler.patched/tcljx/parser/syntax.cljt
--- src/tcljx.compiler/tcljx/parser/syntax.cljt	2025-10-21 15:21:33.118233547 +0200
+++ ../tcljx.compiler.patched/tcljx/parser/syntax.cljt	2025-10-30 07:33:25.525427976 +0100
@@ -1,6 +1,6 @@
 (ns tcljx.parser.syntax
   (:require (tcljx.data [config :as cfg] [wrong :as wrong])
-            [tinyclj.core :as cc])
+            [clojure.core :as cc])
   (:refer-clojure :exclude [vector list map symbol first second]))
 
 (defn info
